components/listing/card/Card.tsx

```diff
@@ -8,38 +8,43 @@ interface CardProps {
   task: Task;
 }
 
-const Card: React.FC<CardProps> = ({ task }) => {
+const Card: React.FC<CardProps> = ({ task: { _id, _user, images, title, tags, createdAt } }) => {
```

Comment:
nie przekazuj całego taska :)
```ts
interface CardProps {
_id: string;
 _user: string; 
itd.
images, title, tags, createdAt
}
```

Reply:
Sporo prościej jest przekazać całego, ale ok. Przerobiłem to w `Card.tsx` i `Cards.tsx`. Jeszcze trzeba nad tym zdjęciem usera pomyśleć, bo w `task` dla `_user` nie ma takiego pola... Ale to już kwestia bazy.